src/tiler.cpp

```diff
@@ -339,12 +341,49 @@ BoundingBox<int> TilerHelper::parseZRange(const std::string &zRange){
     return r;
 }
 
-fs::path TilerHelper::getCacheFolderName(const fs::path &geotiffPath, time_t modifiedTime){
+fs::path TilerHelper::getCacheFolderName(const fs::path &geotiffPath, time_t modifiedTime, int tileSize){
     std::ostringstream os;
-    os << geotiffPath.string() << "*" << modifiedTime;
+    os << geotiffPath.string() << "*" << modifiedTime << "*" << tileSize;
     return Hash::strCRC64(os.str());
 }
 
+fs::path TilerHelper::getFromUserCache(const fs::path &geotiffPath, int tz, int tx, int ty, int tileSize, bool tms, bool forceRecreate){
+    if (std::rand() % 1000 == 0) cleanupUserCache();
```

Comment:
How is this number calculated?

Reply:
Arbitrary. We need to cleanup the cache once in a while without user intervention. Not sure what would be a better strategy. A counter would require shared access to files.